fmn/formatters.py

```diff
@@ -178,7 +176,48 @@ def irc_batch(messages, recipient):
     if len(messages) == 1:
         return irc(messages[0], recipient)
     else:
-        return fedmsg.meta.conglomerate(messages, **config.app_conf)
+        template = u'{subtitle} {delta} {link} {filter_link}\n'
+        formatted_message = u''
+        try:
+            # This apparently returns a list of dicts with the 'link', 'subtitle',
+            # 'subjective', 'secondary_icon' and any number of other keys from the
+            # individual conglomerator.
+            conglomerated = fedmsg.meta.conglomerate(messages, **config.app_conf)
+            for message in conglomerated:
+                formatted_message += template.format(
+                    subtitle=message['subtitle'], delta=message.get('human_time', ''),
+                    link=message['link'], filter_link=_irc_filter_link(recipient))
+        except Exception:
+            _log.exception('An unexpected error occurred while processing an IRC batch')
```

Comment:
Just to be nitpicky, but maybe also log the message IDs in the batch here?

Reply:
Done